<launch>
  
  <node pkg="pg_editor" name="ndt_matching_server" type="ndt_matching_server" output="screen"/>
  <node pkg="pg_editor" name="data_reader_server" type="data_reader_server" output="screen"/>
  <node pkg="pg_editor" name="main_server" type="main_server" output="screen"/> 
  <node pkg="pg_editor" name="tf_broadcaster_node" type="tf_broadcaster_node" output="screen"/>  
  <node pkg="rqt_reconfigure" name="rqt_reconfigure" type="rqt_reconfigure" output="screen"/>  
  <node pkg="rviz" name="rviz" type="rviz" args="-d $(find pg_editor)/rviz/lidar_cal.rviz" output="screen"/>

</launch>
